README.md

```diff
@@ -5,19 +5,26 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/jenkinsci/kubernetes-operator "Go Report Card")](https://goreportcard.com/report/github.com/jenkinsci/kubernetes-operator)
 [![Docker Pulls](https://img.shields.io/docker/pulls/virtuslab/jenkins-operator.svg)](https://hub.docker.com/r/virtuslab/jenkins-operator/tags)
 
-Go to [**our documentation website**](https://jenkinsci.github.io/kubernetes-operator/) for more information, with examples and guides.
-
 ![logo](/assets/jenkins_gopher_wide.png)
 
 ## What's the Jenkins Operator?
 
-Jenkins operator is a Kubernetes native operator which fully manages Jenkins on Kubernetes.
-It was built with immutability and declarative configuration as code in mind.
+The Jenkins Operator is a [Kubernetes Native Operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) which manages operations for Jenkins on Kubernetes.
+It has been built with Immutability and declarative Configuration as Code in mind.
+
+
+## Preliminaries
+
+Considering that this Operator is created for managing instances for Jenkins, 
+it is important to understand what 
```

Comment:
nice, it's clearer now, could you also add the explanation as to way it is important to know this plug-ins?

I think adapting the texts that were removed from the list below, about how operator uses yhe plug-in, could be enough